mlrun/model_monitoring/application.py

```diff
@@ -145,13 +143,14 @@ def run_application(
         :param endpoint_id:             (str) ID of the monitored model endpoint
         :param output_stream_uri:       (str) URI of the output stream for results
 
-        :returns:                       (ModelMonitoringApplicationResult) or
-                                        (list[ModelMonitoringApplicationResult]) of the application results.
+        :returns:                       (ApplicationResult) or
+                                        (list[ApplicationResult]) of the application results.
```

Comment:
I wonder if it's really necessary to duplicate the type annotations in this way.

Reply:
Do you mean the docstring duplication of the type hint in the signature?